### ISSUE TITLE
Clarify how to use existing databases

### ISSUE THREAD

**Issue by esafak** [ponyorm/pony#137]
The examples I saw assumed that the database is created by Pony, but oftentimes there is an existing database we want to interact with. Can you add a simple example?

**Comment by jayrodrigues** [comment]
I fully agree. When connecting to databases for data analysis, we often have to connect to existing databases. Can this be done with Pony ORM?

**Comment by Meowzz95** [comment]
Any updates about this issue..? I'd like to use Pony with an existing database too. Perhaps there's a code generation tool?

**Comment by majidaldo** [comment]
bump

**Comment by Meowzz95** [comment]
> I think a code generation tool is currently an open issue.

Currently since 2015 😂😂😂
